apps/demos/Demos/FileUploader/CustomDropzone/jQuery/index.js

```diff
@@ -8,7 +8,21 @@ $(() => {
     uploadUrl: 'https://js.devexpress.com/Demos/NetCore/FileUploader/Upload',
     visible: false,
     onDropZoneEnter(e) {
```

Comment:
it's possible to use destructuring here
```suggestion
    onDropZoneEnter({ component, dropZoneElement, event }) {
```